conditional/blueprints/dashboard.py

```diff
@@ -89,8 +89,7 @@ def display_dashboard():
             'status': p.status,
             'description': p.description
         } for p in
-        MajorProject.query.filter(MajorProject.uid == member.uid,
-                                  MajorProject.date > start_of_year())]
```

Comment:
The dashboard is meant to display information pertaining to the current evaluations period. With that said, I am not against showing history here, but it should be shown in the proper context. We should either change the styling of past year's major projects (grey them out, etc.) or put them in a new table called "Past Projects" or something of the like.